frameworks/Crystal/amber/setup.sh

```diff
@@ -10,6 +10,8 @@ export GC_MARKERS=1
 
 export AMBER_ENV=production
 
+export DATABASE_URL=postgres://benchmarkdbuser:benchmarkdbpass@TFB-database/hello_world
```

Comment:
Does this not need the port specified?

Reply:
@msmith-techempower We didn't specify database port before but we specify amber port here 👉 https://github.com/TechEmpower/FrameworkBenchmarks/pull/3229/files#diff-2a38afc8380af63cc2e9aea0778f0d6fR53 

@drujensen I'm using `DATABASE_URL` to avoid using encrypted settings (_This is a production environment_), another alternative is using an initializer with this:

```
Granite::ORM.settings.database_url = Amber.settings.database_url
```

WDYT?

Reply:
@msmith-techempower Done! 👍